core/src/components/progress-bar/test/buffer/progress-bar.e2e.ts

```diff
@@ -0,0 +1,25 @@
+import { expect } from '@playwright/test';
+import { configs, test } from '@utils/test/playwright';
+
+configs().forEach(({ title, screenshot, config }) => {
+  test.describe(title('progress-bar: buffer'), () => {
+    test.describe('with a dynamic progress value', () => {
```

Comment:
If we're only making sure the progress bar updates when the value is changed, we can just test this on LTR and one mode, since the other variations are already checked visually in the `basic` test. Ditto for `determinate` below.